app/Support/MaintenanceService.php

```
@@ -487,6 +496,15 @@ public function checkExpiredReservations(): int
                 // Invia notifiche differite DOPO il commit della transazione
                 $reassignmentService->flushDeferredNotifications();
 
+                // Notifica l'utente che la sua prenotazione è scaduta (GAP-2),
+                // stesso pattern di checkExpiredPickups (email fuori transazione).
+                try {
+                    $notificationService = new NotificationService($this->db);
+                    $notificationService->sendReservationExpiredNotification($id);
+                } catch (\Throwable $e) {
+                    SecureLogger::warning('Reservation expired notification failed', ['prestito_id' => $id, 'error' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Support\SecureLogger' in method 'checkExpiredReservations'. (undefined)

(StaticAccess)